blog/Community/2015-03_11_Announce_of_the_mailing_lists.md

```diff
@@ -0,0 +1,16 @@
+Title: Announce of the mailing lists
+Date: 2015-03-11
+Comments_at: https://github.com/FreshRSS/freshrss.org/issues/21
+
+Did you know that FreshRSS has two mailing lists since few weeks?
+
+In order to welcome our users and developers, we have decided to open them so we can discuss about FreshRSS without forcing the use of GitHub and still keeping a community discussion.
+
+- **mailing@freshrss.org** is mainly for users: if you have questions and you're not sure if it's interesting to ask them on GitHub, you can join us on this mailing. We'll also announce development progress once a month so you'll always know what will be the next great FreshRSS features.
+- **dev@freshrss.org** is destined to developers only. If you want to know more about the source code, what are the libraries included in FreshRSS, how do pull requests work, etc. this is the place to be!
+
+Mailing lists are hosted on our server and you can subscribe to one or both of them by following [this link](https://freshrss.org/mailman/listinfo). Archives are public so you are able to see what have already been written, especially the “[welcome-message](http://freshrss.org/pipermail/mailing/2015-March/000015.html)”.
+
+Note that if you want to participate (i.e. send mails on mailings) it is required to subscribe to them. English is the language to prefer but since FreshRSS has a great French-community, it may happen that French is used instead. If you want to participate to such discussions, don't hesitate to ask to continue in English.
+
+We wait you!
```

Comment:
I think that `We wait you` is not correct. I think `We are waiting for you` is more correct. To be verified!